cli/build.gradle.kts

```diff
@@ -44,14 +44,7 @@ dependencies {
 
 val shadowJar = tasks.named<com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar>("shadowJar") {
   // We set the Shadow Jar to have NO classifier, making it the "Main" artifact
-  archiveClassifier.set("")
   mergeServiceFiles()
-  minimize {
```

Comment:
I think you need to add more to this section and leave everything else alone. 

minimize is what is removing the classes we use, so add them back here until it works!